packages/components/src/__actions__/Button/v3/_docs/Button.spec.stories.tsx

```diff
@@ -26,38 +21,44 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const ButtonDefault: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>
+export const TriggerPending: Story = {
+  render: ({ children, isPending = false, ...otherProps }) => {
+    const [isPendingStatus, setIsPendingStatus] =
+      React.useState<boolean>(isPending)
+
+    return (
       <>
-        {children}
-        <AddIcon role="presentation" />
+        <Button
+          {...otherProps}
+          isPending={isPendingStatus}
+          pendingLabel="loading"
+          onPress={() => {
+            setIsPendingStatus(true)
+            setTimeout(() => setIsPendingStatus(false), 3000)
+          }}
+        >
+          {children}
+        </Button>
       </>
-    </Button>
-  ),
-}
-
-export const IsHovered: Story = {
-  ...ButtonDefault,
-  play: async ({ canvasElement, step }) => {
+    )
+  },
+  play: async ({ canvasElement }) => {
     const canvas = within(canvasElement.parentElement!)
     const button = canvas.getByRole("button")
-
-    await step("Hover shows", async () => {
-      await userEvent.hover(button)
-      await expect(button).toHaveAttribute("data-hovered", "true")
-    })
-  },
-}
-export const IsHoveredReversed: Story = {
-  ...IsHovered,
-  parameters: {
-    reverseColors: true,
+    expect(button).toHaveAccessibleName("Label")
+    await button.focus()
+    await userEvent.click(button)
+    expect(button).toHaveAccessibleName("loading")
   },
 }
 
-export const IsFocused: Story = {
-  ...ButtonDefault,
+export const ButtonWithTooltip: Story = {
```

Comment:
This would belong better in Tooltip stories since the dependency is the other way around